src/about.html

```diff
@@ -40,54 +40,54 @@ <h3>What are Existential Graphs?</h3>
         <dl>
-          <dt>Sheet of Assertion</dt>
-          <dd>The blank page on which the graph is constructed.</dd>
+          <dt>The Sheet of Assertion</dt>
+          <dd>Blank page on which the graph is constructed. May be referred to as "The Sheet."</dd>
 
           <dt>Atoms</dt>
-          <dd>The letters or phrases which are used to represent the propositions.</dd>
+          <dd>Letters or phrases used to represent propositions. Peirce My Heart only supports letters.</dd>
 
           <dt>Cuts</dt>
           <dd>
-            Fully closed curves that are used to represent negation. Everything contained within 
-            the cut will be considered to be negated. Cuts can be nested and concatenated with 
-            other cuts. Empty cuts, i.e. cuts that do not contain anything within them, are 
-            allowed to be present on the graph. Each cut represents a "level" in 
-            the graph; a nested cut represents one level deeper than the level of its parent cut.
+            Fully closed curves representing negations. Every atom and cut contained within 
+            some other cut is considered negated. Cuts can be nested within
+            other cuts. Empty cuts, i.e. cuts that do not contain anything, are 
+            allowed to exist on The Sheet. Each cut corresponds to an integer cut "level" in 
+            the graph, and a nested cut corresponds to one cut level deeper than the level of its parent.
           </dd>
         </dl>
 
         <h4>How AEGs Work</h4>
-        Anything drawn on this sheet is asserted to be <b>True</b>. Phrases, subgraphs and even the 
-        entire graph can be equated to <b>True</b> or <b>False.</b> <br/>
-        Since everything drawn on the sheet is considered <b>True</b>, placing multiple atoms on the 
-        same levels can be considered as a conjunction between the atoms. This further applies to 
-        atoms and cuts on the same level as well (Note: Asserting that multiple propositions 
-        are True all at the same time in turn holds that the conjunction of those propositions 
-        is True as well). <br/>
-        Everything contained within a cut is considered negated. As such, an empty cut represents
+        Anything drawn on The Sheet is asserted as <b>True</b>. Phrases, subgraphs and The 
+        Sheet itself may evaluate to <b>True</b> or <b>False.</b> <br/>
+        Since everything drawn on The Sheet is <b>True</b>, placing multiple atoms on the 
+        same level may be considered creating a conjunction between these atoms. This further applies to 
+        atoms and cuts on the same level (Note: Asserting that multiple propositions 
+        are <b>True</b> all at the same time means that the conjunction of these propositions 
+        is <b>True</b> as well). <br/>
+        Because everything contained within a cut is negated, and The Sheet corresponds to <b>True</b>, an empty cut corresponds to
```

Comment:
You can't start a sentence with "Because"

Reply:
Yes I can!